packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -8,9 +8,28 @@ import PropTypes from 'prop-types';
  * This component expects all necessary scripts for BookReader to be pre-loaded onto the page before use.
  * Please see https://github.com/internetarchive/bookreader for more instructions & examples.
  *
+ * If jsia is specified then it adds Internet Archive specific stuff via the BookreaderJSIA function
+ * the script BookreaderJSIA should previously have been loaded.
+ *
+ * TODO - someone could merge all or part of BookreaderJSIA into here and obsolete BookreaderJSIA ideally without changing the interface
+ * TODO - except Isa doesnt want the Archive's lending logic in here.
```

Comment:
Up to you - - I like to leave comments in code about things that we know can or should be done in future, as it makes it more likely someone revising it in future will pick that up.  I also thought the reference to you was relevant, so that someone didn't just think it was a good idea when you had (completely reasonable, but non obvious) objections to putting the JSIA code here.  If you prefer, could revise to ...

TODO - The BookreaderJSIA code cant go in here, because it is InternetArchive specific, future development could be a wrapper, for this, that includes BookreaderJSIA

Reply:
updated